<?xml version="1.0"?>
<launch>
  <node name="extended_toe_finding_and_yolo" pkg="intersection_recognition" type="extended_toe_finding_and_yolo" output="screen" >
    <remap from="scan"            to="merged_scan"/>
    <remap from="image_rae"       to="image/mercator"/>
    <param name="SCAN_HZ"         value="10"/>
    <param name="distance_thresh" value="3.0"/>
    <param name="door_size_thresh" value="0.5"/>
  </node>
  <node name="rviz" pkg="rviz" type="rviz" />

</launch>
